### PR TITLE
Array reverse

### PR DESCRIPTION
**Pull request by easleyjs**
- [x ] Top-level README “Table of Contents” is updated
 - [x ] README for this challenge is complete
       - [x ] Summary, Description, Approach & Efficiency, Solution
       - [x ] Picture of whiteboard
       - [x ] Link to code
 - [x ] Feature tasks for this challenge are completed
 - [x ] Unit tests written and passing
       - [x ] “Happy Path” - Expected outcome
       - [ ] Expected failure
       - [ ] Edge Case (if applicable/obvious)

### CODE REVIEW

File: javascript/array-reverse/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Challenge Title
+## Reversing an array
+
+## Whiteboard Process
+![](whiteboard.png)
```
Comment (arpatterson31) [inline javascript/array-reverse/README.md:5]: Hey Joshua! Great job with the first code challenge. You'll want to make sure you're hitting all sections in your whiteboard process. One of the most important sections is the algorithm because it will show the interview how you break down and solve problems step by step!
